db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150308012120) do
+ActiveRecord::Schema.define(version: 20150503031721) do
```

Comment:
[rubocop] Separate every 3 digits in the integer portion of a number with underscores(_).